src/mutator_test.cc

```diff
@@ -678,12 +678,23 @@ TYPED_TEST(MutatorTypedTest, Serialization) {
 }
 
 TYPED_TEST(MutatorTypedTest, UnknownFieldTextFormat) {
+  // NOTE: This test has little chance of passing when method
+  //       TextFormat::Parser::AllowUnknownField is not available
+#if GOOGLE_PROTOBUF_VERSION >= 3008000  // commit 176f7db11d8242b36a3ea6abb1cc436fca5bf75d of >=3.8.0
```

Comment:
Do we really need before 3.8.0?

Reply:
@vitalybuka yes, see https://packages.ubuntu.com/focal/libprotobuf-dev